xml/art_pci-dss.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook50-profile.xsl"
+ type="text/xml"
+ title="Profiling step"?>
+
+<!DOCTYPE article
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+  %entities;
+  <!ENTITY productnamex "&sls;">
+  <!ENTITY productnumberx "&product-ga; SP&product-sp;">
+]>
+
+<article xml:id="article-pci-dss" xml:lang="en" version="5.0"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>&sle_pcidss_guide;</title>
+ <titleabbrev>&sle_pcidss_guide;</titleabbrev>
+ <info>
+  <productname>&productnamex;</productname>
+  <productnumber>&productnumberx;</productnumber>
+  <date><?dbtimestamp format="B d, Y" ?></date>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker>
+    <dm:url>https://bugzilla.suse.com/enter_bug.cgi</dm:url>
+    <dm:component>Documentation</dm:component>
+    <dm:product>SUSE Linux Enterprise Server 15 SP4</dm:product>
+    <dm:assignee>taroth@suse.com</dm:assignee>
+   </dm:bugtracker>
+   <dm:editurl>https://github.com/SUSE/doc-sle/edit/main/xml/</dm:editurl>
+   <dm:translation>no</dm:translation>
+  </dm:docmanager>
+  <!-- abstract: -->
+  <xi:include xpointer="element(/1/2/1)" href="security_pcidss.xml"/>
+ </info>
+ <!-- intro paras: -->
+ <xi:include xpointer="element(/1/3)" href="security_pcidss.xml"/>
+ <xi:include xpointer="element(/1/4)" href="security_pcidss.xml"/>
+ <xi:include xpointer="element(/1/5)" href="security_pcidss.xml"/>
+ <xi:include xpointer="element(/1/6)" href="security_pcidss.xml"/>
+
+ <!-- sections: -->
+ <xi:include xpointer="element(/1/7)" href="security_pcidss.xml"/>
+ <xi:include xpointer="element(/1/8)" href="security_pcidss.xml"/>
+ <xi:include xpointer="element(/1/9)" href="security_pcidss.xml"/>
+
```

Comment:
not sure about all of these xpointers. I'm finding a few typos and tagging errors, I will do a thorough proofread in a separate PR.